sdk/ml/azure-ai-ml/azure/ai/ml/operations/_workspace_operations.py

```diff
@@ -469,6 +476,61 @@ def _populate_arm_paramaters(self, workspace: Workspace) -> Tuple[dict, dict, di
                 group_name,
             )
         else:
+            # if workspace is located in a region where app insights is not supported, we do this swap
+            if workspace.location in ["westcentralus", "eastus2euap", "centraluseuap"]:
+                app_insights_location = "southcentralus"
+            else:
+                app_insights_location = workspace.location
+            # check if default resource group already exists
+            rg_is_existing = default_resource_group_for_app_insights_exists(
+                self._credentials,
+                self._subscription_id,
+                app_insights_location)
```

Comment:
could you run `black` on each of the files in your PR? `black` is a python code formatter, and it will keep the style consistent with the rest of the SDK. It's on pypi: https://pypi.org/project/black/, and after installing it, you can just run:

```
black <path to file> --line-length=120
```

Reply:
I have done this for all files except for the arm template file. It does not make sense for the ARM template for readability.